Helper/Webhook.php

```diff
@@ -1348,7 +1348,7 @@ private function isAutoCapture($notificationPaymentMethod): bool
 
             // if auto capture mode for openinvoice is turned on then use auto capture
             if ($captureModeOpenInvoice &&
-                $this->adyenHelper->isPaymentMethodOpenInvoiceMethodValidForAutoCapture($notificationPaymentMethod)
+                $this->adyenHelper->isPaymentMethodOpenInvoiceMethod($notificationPaymentMethod)
```

Comment:
@RokPopov 
Does a payment method always support `AutoCapture` if it is an `OpenInvoice` method?

Reply:
Double-checking this, as of now this is the case - we support `AutoCapture` for all `OpenInvoice` payment methods